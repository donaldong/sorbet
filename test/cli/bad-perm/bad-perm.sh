#!/bin/bash

dir=$(mktemp -d)
echo "T.reveal_type(1)" > "$dir/file.rb"
mkdir "$dir/no-read"
chmod 000 "$dir/no-read"
main/sorbet --silence-dev-message "$dir" 2>&1
