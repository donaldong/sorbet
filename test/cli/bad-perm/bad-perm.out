file.rb:1: `T.reveal_type` can only reveal types in `# typed: true` files (or higher) https://srb.help/5039
     1 |T.reveal_type(1)
        ^^^^^^^^^^^^^^^^
Errors: 1
